ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicationManager.java

```diff
@@ -77,6 +78,7 @@ public VcrReplicationManager(CloudConfig cloudConfig, ReplicationConfig replicat
     this.cloudStorageCompactor =
         cloudConfig.cloudBlobCompactionEnabled ? new CloudStorageCompactor(cloudDestination, cloudConfig,
             partitionToPartitionInfo.keySet(), vcrMetrics) : null;
+    trackPerPartitionLagInMetric = replicationConfig.replicationTrackRemoteFromLocalPerDatacenterLag;
```

Comment:
looks like this variable name may need adjusting since it is assigned to the per datacenter metric config

Reply:
ah, good catch, will change. 

For naming, on second thought, I probably won't change the name of `replicationTrackPerPartitionLagFromRemote` (this is introduced by @zzmao ). As you can see, I renamed it in this PR but since it requires to change config name/ sensor name in closed source, I feel like I would change it back. What do you think? @cgtz @zzmao

Reply:
It probably isn't used in too many places and isn't a required config so it may be okay to change if you think the new names are significantly more clear. Of course, you may need to keep both configs in src files temporarily so that it works before and after the deployment of this change and then remove the old config after this change is fully deployed. Up to you if you want to do this.